cli.js

```diff
@@ -44,6 +44,9 @@ if (argv.ts) {
     docs = docs.use(/\.tsx?$/, new TypescriptPlugin({
         excludePaths: ["__tests__"],
     }));
+    docs = docs.use(/\.tsx?$/, new TypedocPlugin({
+        excludePaths: ["__tests__"],
+    }));
```

Comment:
well the old one works with @blueprintjs/docs, but the new one does not have the same format because it's not only interfaces.